### PR TITLE
Fix/proposer factory

### PR DESCRIPTION
**Pull request by antiyro**
removing unecessary config and concensus from substrate

### CODE REVIEW

File: crates/client/db/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//! A database backend storing data about madara chain
+//! A database backend storing data about deoxys chain
```
Comment (jbcaron) [inline crates/client/db/src/lib.rs:1]: starknet chain

File: crates/client/rpc/src/methods/read/estimate_fee.rs [hunk 2]
```diff
@@ -77,7 +77,7 @@ where
 
     let estimates = fee_estimates
             .into_iter()
-			// FIXME: https://github.com/keep-starknet-strange/madara/issues/329
+			// FIXME: https://github.com/kasarlabs/deoxys/issues/329
```
Comment (jbcaron) [inline crates/client/rpc/src/methods/read/estimate_fee.rs]: issue does not match

File: crates/node/src/commands/run.rs [hunk 3]
```diff
@@ -182,9 +182,7 @@ pub fn run_node(mut cli: Cli) -> Result<()> {
         fetch_block_config.sound = cli.run.sound;
         fetch_block_config.verify = !cli.run.disable_root;
         fetch_block_config.api_key = cli.run.gateway_key.clone();
-
         update_config(&fetch_block_config);
-        log::debug!("Using fetch block config: {:?}", fetch_block_config);
```
Comment (jbcaron) [inline crates/node/src/commands/run.rs:187]: debug can be helpful

File: crates/primitives/convert/src/contract.rs [hunk 4]
```diff
@@ -31,7 +31,7 @@ pub fn from_rpc_contract_class(contract_class: ContractClassCore) -> anyhow::Res
 // TODO: implement this
 pub fn to_contract_class_sierra(_: &ContractClassV1, abi: String) -> anyhow::Result<ContractClassCore> {
     Ok(ContractClassCore::Sierra(FlattenedSierraClass {
-        sierra_program: vec![], // FIXME: https://github.com/keep-starknet-strange/madara/issues/775
+        sierra_program: vec![], // FIXME: https://github.com/kasarlabs/deoxys/issues/775
```
Comment (jbcaron) [inline crates/primitives/convert/src/contract.rs]: issue does not match

File: crates/runtime/src/pallets.rs [hunk 5]
```diff
@@ -119,7 +119,7 @@ impl frame_system::Config for Runtime {
 // We want to support multiple flavors of consensus algorithms.
 // Specifically we want to implement some proposals defined in the Starknet community forum.
 // For more information see: https://community.starknet.io/t/starknet-decentralized-protocol-i-introduction/2671
-// You can also follow this issue on github: https://github.com/keep-starknet-strange/madara/issues/83
+// You can also follow this issue on github: https://github.com/kasarlabs/deoxys/issues/83
```
Comment (jbcaron) [inline crates/runtime/src/pallets.rs]: issue does not match

**Review by jbcaron — CHANGES_REQUESTED** [review]
refer to comments

**Review by jbcaron — APPROVED** [review]

**Review by Trantorian1 — APPROVED** [review]

**Pull request merged**